ctlra/devices/ni_kontrol_z1.c

```diff
@@ -52,36 +52,70 @@ struct ni_kontrol_z1_ctlra_t {
 };
 
 static const char *ni_kontrol_z1_names_sliders[] = {
-	"Gain (Left)",
-	"Eq High (Left)",
-	"Eq Mid (Left)",
-	"Eq Low (Left)",
-	"Filter (Left)",
-	"Gain (Right)",
-	"Eq High (Right)",
-	"Eq Mid (Right)",
-	"Eq Low (Right)",
-	"Filter (Right)",
+	"Gain (L)",
+	"Eq High (L)",
+	"Eq Mid (L)",
+	"Eq Low (L)",
+	"Filter (L)",
+	"Gain (R)",
+	"Eq High (R)",
+	"Eq Mid (R)",
+	"Eq Low (R)",
+	"Filter (R)",
 	"Cue Mix",
-	"Fader (Left)",
-	"Fader (Right)",
+	"Fader (L)",
+	"Fader (R)",
 	"Crossfader",
 };
 #define CONTROL_NAMES_SLIDERS_SIZE (sizeof(ni_kontrol_z1_names_sliders) /\
```

Comment:
Might be easier to read with the linebreak before the sizeofs (otherwise it looks at first glance like a /\ operator :)

Reply:
I do line-breaks at any operator, but leave as much as can on the first line. So in this case, although it looks weird, it is mostly consistent in the code-base. Agree that a space betweeen / and \ would be good. \m/

Reply:
Makes sense, I'm all for consistency.